src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -6748,8 +6757,7 @@ def test_arg_discontinuous(monkeypatch, annexed):
 
     # 1 discontinuous field returns true
     # Check w3, wtheta and w2v in turn
-    import psyclone.config
-    monkeypatch.setattr(psyclone.config, "COMPUTE_ANNEXED_DOFS", annexed)
+    monkeypatch.setattr(_CONFIG, "_compute_annexed_dofs", annexed)
```

Comment:
Create one here rather than at top of the file? There could be side effects between different tests in parallel?

Reply:
Monkeypatch will keep things local.